### ISSUE TITLE
KarmaBot goes to sleep a lot

### ISSUE THREAD

**Issue by alanbsmith** [alanbsmith/karma-bot#1]
It would be nice if KarmaBot would either stay awake or restart itself.

**Comment by tinynumbers** [comment]
Is this deployed on Heroku?

**Comment by alanbsmith** [comment]
Yeah, but I'm not sure if it shuts down because the server is sleeping, or if the websocket closes after some timeout.
